builtin/credential/okta/path_login.go

```diff
@@ -87,7 +95,27 @@ func (b *backend) pathLoginRenew(
 		return nil, fmt.Errorf("policies have changed, not renewing")
 	}
 
-	return framework.LeaseExtend(0, 0, b.System())(req, d)
+	ttl, maxTTL, err := b.getTTLs(req)
+	if err != nil {
+		return nil, err
+	}
+
+	return framework.LeaseExtend(ttl, maxTTL, b.System())(req, d)
+}
+
+func (b *backend) getTTLs(req *logical.Request) (ttl, maxTTL time.Duration, err error) {
+
+	cfg, err := b.Config(req.Storage)
+	if err != nil {
+		return 0, 0, err
+	}
+	if cfg == nil {
+		return 0, 0, errors.New("Okta backend not configured")
+	}
+
+	ttl, maxTTL, err = b.SanitizeTTLStr(cfg.TTL.String(), cfg.MaxTTL.String())
```

Comment:
This call is unnecessary, you're converting from a duration to a string back to a duration. `LeaseExtend` does bounds checking so you'll be fine as-is.